packages/rum-core/src/common/patching/index.js

```diff
@@ -46,10 +47,10 @@ function patchAll() {
     patchHistory(function(event, task) {
       patchEventHandler.send(HISTORY, [event, task])
     })
-
     patchEventTarget(function(event, task) {
       patchEventHandler.send(EVENT_TARGET, [event, task])
     })
+    state.patchedTime = now()
```

Comment:
Let's move this to bootstrap and use `bootstrapTime`, it's more generic that way.

Reply:
Sure, will do that.